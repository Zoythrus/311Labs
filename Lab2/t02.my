mdfulness somethg brg experiencg md beg mdful. growg showg mdful, remodelg bra. 

beg bra. brg experiencg growg md mdful, mdful. mdfulness remodelg showg somethg 
