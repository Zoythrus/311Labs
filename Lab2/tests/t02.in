in
While mindfulness is something we all naturally possess, it is more readily available to us when we practice on a daily basis. 
Whenever you bring awareness to what you are directly experiencing via your senses, or to your state of mind via your thoughts and emotions, you are being mindful. 
And there is growing research showing that when you train your brain to be mindful, you are actually remodeling the physical structure of your brain.
