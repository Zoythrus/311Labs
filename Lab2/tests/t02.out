beg bra bra. brg experiencg growg md mdful, mdful. mdfulness remodelg showg somethg tra 
